src/test/java/seedu/trackermon/logic/LogicManagerTest.java

```diff
@@ -76,13 +79,13 @@ public void execute_validCommand_success() throws Exception {
 
     @Test
```

Comment:
Where is the header for the public test methods?

Reply:
This PR is only for header comments, public test methods include a lot of other test cases, i'll create a new PR for that

Reply:
Noted